packages/api/src/controllers/insightHelpers.js

```diff
@@ -236,105 +238,100 @@ export const getBiodiversityAPI = async (pointData, countData) => {
     data: [],
   };
 
-  const dictionary = {
-    Aves: 'Birds',
-    Insecta: 'Insects',
-    Plantae: 'Plants',
-    Amphibia: 'Amphibians',
-    Mammalia: 'Mammals',
+  const filterDictionary = {
+    Birds: { classKey: 212 },
+    Insects: { classKey: 216 },
+    Plants: { kingdomKey: 6 },
+    Amphibians: { classKey: 131 },
+    Mammals: { classKey: 359 },
```

Comment:
Where are these keys from?

Reply:
Unfortunately I couldn't find anywhere in the docs where the keys are explicitly specified -- I ended up doing several queries to the API directly (e.g. https://api.gbif.org/v1/occurrence/search?taxonKey=1) and figuring out from the responses which class or kingdom keys matched the class or kingdom names we were previously using